.github/workflows/ci.yml

```
@@ -31,6 +31,11 @@ jobs:
         run: cargo build --verbose --all
       - name: cargo test
         run: cargo test --verbose --all
+      - name: cargo clean release builds
+        run: cargo clean -p ikura-node --release && \
+             cargo clean -p ikura-shim --release && \
+             cd demo/sovereign && \
+             cargo clean -p sov-demo-rollup --release && \
```

```suggestion
             cargo clean -p sov-demo-rollup --release
```

dangling \. Did you forget a line there?

Do you think it would be better to use `--manifest-path` instead of `cd`?

---

> Did you forget a line there?

Absolutely, I forgot to remove the last characters in the last line. I was also convinced that I had created this PR as a draft to test if the CI was okay, but I didn't 😅

> Do you think it would be better to use --manifest-path instead of cd?

great call, I update it as you suggested!!